src/screens/ChannelListScreen/elements/CustomPreviewMessage.js

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import {ChannelPreviewMessage} from 'stream-chat-react-native';
 import {StyleSheet, Text} from 'react-native';
 import {Context} from '../../../context';
-import {COLORS} from '../../../utils/theme';
 
 const styles = StyleSheet.create({
   message: {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for the color theme has been removed, and no replacement has been provided. This could lead to potential issues if the color theme is used elsewhere in the code. If the color theme is not being used, this change is fine.



<!-- This is an auto-generated comment by OSS CodeRabbit -->